### PR TITLE
fix(emailVerification): fixed email verification

### PR DESCRIPTION
**Pull request by simran142002**
### Functionality:
On clicking on verify any number of times member will get added only once to the team

### Solution:
Made changes in user verification component

### Risk level:
- [ ] high 
- [x] medium
- [ ] low

### How to test:
ng serve -o
firebase emulators:start

### CODE REVIEW

**Comment by viv9k** [comment]
I need an explanation what's going on here

**Comment by simran142002** [comment]
> 

okay sure

**Review by Pendyala-thanvi — APPROVED** [review]

File: functions/model/users/tark/checkIfUserAlreadyAMember.js [hunk 1]
```diff
@@ -0,0 +1,27 @@
+const {getTeam} = require("../../teams/lib");
+
+exports.checkIfUserAlreadyAMember = function(request, response) {
+  const organizationDomain = request.body.data.OrganizationDomain;
```
Comment (viv9k) [inline functions/model/users/tark/checkIfUserAlreadyAMember.js:4]: eslint issues

**Review by viv9k — APPROVED** [review]
Approved

**Pull request merged**